bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -65,10 +56,11 @@ spec:
       {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 6 }}
       {{- end }}
     {{- end }}
-    type: {{ .Values.grafana.service.type }}
-    {{- if .Values.grafana.service.extraPorts }}
-    ports: {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.service.extraPorts "context" $) | nindent 6 }}
-    {{- end }}
+    spec:
+      type: {{ .Values.grafana.service.type }}
+      {{- if .Values.grafana.service.extraPorts }}
+      ports: {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.service.extraPorts "context" $) | nindent 6 }}
+      {{- end }}
   {{- $imagePullSecrets := include "common.images.pullSecrets" (dict "images" (list .Values.operator.image .Values.grafana.image .Values.grafana.pluginsInitContainerImage) "global" .Values.global) }}
```

Comment:
If I'm not mistaken the `.Values.grafana.image` and `.Values.grafana.pluginsInitContainerImage` are no longer used. Please review the chart's templates to remove the unused references.

Reply:
I just removed